client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -19,7 +19,8 @@ interface ImageDataItemInternal extends ImageDataItem {
 
 function loadImageFunc(imageDataItem: ImageDataItem, img: HTMLImageElement) {
   // eslint-disable-next-line no-param-reassign
-  img.src = imageDataItem.url;
+  img.src = '/api/v1/dive_dataset/61afbcacdbf9dc586bd7e27c/media/6196c871555dd52064119464/download';
+  // img.src = imageDataItem.url;
```

Comment:
BTW, you can avoid editing the client by returning the new URLS  from the `crud_dataset.get_media` function.  Instead of the existing girder API download URIs, you can return the new one you made.

Reply:
great, that is the plan for wrapping this up. This line was leftover from a quick check and would change back on cleanup/finishing

Reply:
fixed in 17232d5104da398953d5baf748e56cfaa06183f9